src/main/java/com/monumental/controllers/MonumentController.java

```diff
@@ -211,21 +217,53 @@ public MonumentBulkValidationResult validateMonumentCSV(@ModelAttribute BulkCrea
     }
 
     /**
-     * Create monuments from csv or zip
+     * Start the job to create monuments from csv or zip
      * @param request - Contains the field mapping and the file to process
-     * @return BulkCreateResult - Object representing the results of the Bulk Monument Create operation
+     * @return AsyncJob - Object containing the Id of the job created and the current value of the Future object
      */
-    @PostMapping(value = "/api/monument/bulk/create")
-    public List<Monument> createMonumentCSV(@ModelAttribute BulkCreateMonumentRequest request) throws IOException {
+    @PostMapping("/api/monument/bulk/create/start")
+    public AsyncJob startBulkCreateMonumentJob(@ModelAttribute BulkCreateMonumentRequest request) throws IOException {
         BulkCreateMonumentRequest.ParseResult parseResult = request.parse(this.monumentService);
 
         MonumentBulkValidationResult validationResult = this.monumentService.validateMonumentCSV(
                 parseResult.csvContents, parseResult.mapping, parseResult.zipFile
         );
 
-        return this.monumentService.bulkCreateMonuments(
+        /* TODO: This is not a particularly easy way of creating AsyncJobs, I can't think of a way to abstract
+         * it away currently because the AsyncJob must be passed to the CompletableFuture method, and the CompletableFuture
+         * must be passed to the AsyncJob, making it difficult to do so dynamically
+         */
+        AsyncJob job = this.asyncMonumentService.createJob();
+        job.setFuture(this.asyncMonumentService.bulkCreateMonuments(
+                job,
                 new ArrayList<CsvMonumentConverterResult>(validationResult.getValidResults().values())
-        );
+        ));
+        return job;
```

Comment:
Here we:

* Create an empty `AsyncJob`. At this point all it has is a unique Id, and is stored within our `AsyncMonumentService` so that we can retrieve it by Id in later requests.
* Run the `bulkCreateMonuments` method, which returns an **incomplete** `CompletableFuture`. Think of this as an unresolved promise in Javascript.
* Pass the `AsyncJob` into the `bulkCreateMonuments` method so that it can update its progress as it runs

So, what gets returned here looks like this: 
```
{
    "id": 1,
    "future": {
        "done": false,
        "cancelled": false,
        "completedExceptionally": false,
        "numberOfDependents": 0
    },
    "progress": 0.0
}
```